### PR TITLE
validate diskUsageThreshold and diskUsageLwmThreshold

### PR DESCRIPTION
**Pull request by wenbingshen**
### Motivation

When `diskUsageThreshold < diskUsageLwmThreshold`, the bookie can be started normally. When the disk usage reaches `diskUsageThreshold` , bookie will automatically switch to `ReadOnly` mode. The `LedgerDirsMonitor` then switches the bookie back to  `read-write` mode since the disk usage is less than `diskUsageLwmThreshold`, the bookie will switch state back and forth frequently.

### Changes
When creating `LedgerDirsMonitor`, we need to validate `diskUsageThreshold` and `diskUsageLwmThreshold` first.

### CODE REVIEW

**Comment by wenbingshen** [comment]
rerun failure checks

**Comment by wenbingshen** [comment]
rerun failure checks

**Review by hangc0276 — APPROVED** [review]

**Comment by wenbingshen** [comment]
@dlg99 @eolivelli @zymap @Shoothzj Can you help take a look at this pr? Thanks.

**Comment by hangc0276** [comment]
rerun failure checks

**Comment by wenbingshen** [comment]
rerun failure checks

**Comment by wenbingshen** [comment]
rerun failure checks

**Comment by wenbingshen** [comment]
rerun failure checks

**Comment by wenbingshen** [comment]
rerun failure checks

**Comment by wenbingshen** [comment]
rerun failure checks

**Comment by wenbingshen** [comment]
rerun failure checks

**Comment by wenbingshen** [comment]
rerun failure checks

**Comment by wenbingshen** [comment]
rerun failure checks

**Comment by wenbingshen** [comment]
rerun failure checks

**Comment by wenbingshen** [comment]
rerun failure checks

File: bookkeeper-server/src/main/java/org/apache/bookkeeper/bookie/LedgerDirsMonitor.java [hunk 1]
```diff
@@ -229,5 +231,18 @@ private void checkDirs(final LedgerDirsManager ldm)
         }
         ldm.getWritableLedgerDirs();
     }
+
+    @VisibleForTesting
+    void setDiskSpaceThreshold(float diskSpaceThreshold, float diskUsageLwmThreshold) {
```
Comment (Shoothzj) [inline bookkeeper-server/src/main/java/org/apache/bookkeeper/bookie/LedgerDirsMonitor.java:236]: do we really need this method? can we instantiate a LedgerDirsMonitor to run tests?

Comment (wenbingshen) [reply bookkeeper-server/src/main/java/org/apache/bookkeeper/bookie/LedgerDirsMonitor.java]: Make sense to me. I have addressed your comments. PTAL.

**Comment by wenbingshen** [comment]
rerun failure checks

**Comment by wenbingshen** [comment]
rerun failure checks

**Comment by wenbingshen** [comment]
rerun failure checks

**Review by Shoothzj — APPROVED** [review]

**Pull request merged**